app/celery/tasks.py

```diff
@@ -118,13 +122,10 @@ def process_job(job_id, sender_id=None):
 
     current_app.logger.debug("Starting job {} processing {} notifications".format(job_id, job.notification_count))
 
-    for row in RecipientCSV(
-            s3.get_job_from_s3(str(service.id), str(job_id)),
-            template_type=template.template_type,
-            placeholders=template.placeholders,
-            max_rows=get_csv_max_rows(service.id),
-    ).get_rows():
-        process_row(row, template, job, service, sender_id=sender_id)
+    csv = get_recipient_csv(job, template)
+    queue_override = queue_to_use(len(csv))
```

Comment:
Think you don't need to pass the queue here. Consider moving the logic to `process_job` and use `job.notification_count`.

Reply:
Ah this is great, I didn't notice that the `job` already have a notification count. I will just double-check to see how it is stored to make sure it is correct before making the change.

Reply:
I spoke with Antoine offline and we resolved to lower the default to `200` entries in a CSV for the bulk queue redirection to activate.